ui/nuxeo-data-table/data-table-settings.js

```diff
@@ -19,6 +19,7 @@ import '../nuxeo-button-styles.js';
         <style include="nuxeo-button-styles">
           :host {
             position: relative;
+            width: 30px;
```

Comment:
I would be more comfortable with using a `min-height` here instead of hard coding the value. Do you think it would work in this context?

Reply:
You mean `min-width` @semisse?

Reply:
sure, we can use min-width :)

Reply:
Yes, I wanted to say min-width 🤯  thanks, Andrea